Misc/NEWS.d/next/Library/2020-06-06-14-09-55.bpo-33689.EFUDH7.rst

```diff
@@ -0,0 +1,2 @@
+Added support for empty .pth lines in the :mod:`site.py`

```

Comment:
Since this is a bugfix rather than a new feature, I'd re-word this as something like "ignore empty or whitespace-only lines".

Reply:
Also mention that this is what the documentation already says.

Reply:
Fixed to:
Ignore empty or whitespace-only lines according to the existing documentation.